### PR TITLE
Add drone plugin

### PR DESCRIPTION
**Pull request by sasso**
Finds the latest successful build in a specified repository and creates a new build.

### CODE REVIEW

File: plugins/drone/drone.go [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+		}
+
+		for _, _build := range builds {
+			if _build.Status == "success" && _build.Ref == fmt.Sprintf("refs/heads/%s", c.Branch) {
+				return _build, nil
+			}
+		}
+
+		return Build{}, errors.New("No successful build found")
+	} else {
```
Comment (amoskyler) [inline plugins/drone/drone.go]: style preference, you don't need the else.

File: plugins/drone/drone.go [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+				log.ErrorWithFields(err.Error(), log.Fields{
+					"hash": payload.Release.HeadFeature.Hash,
+				})
+
+				x.events <- e.NewEvent(transistor.GetAction("status"), transistor.GetState("failed"), err.Error())
+				return nil
+			}
+
+			// Create a new build
+			build, err := startBuild(e, droneConfig, successfulbuild.Number)
```
Comment (amoskyler) [inline plugins/drone/drone.go]: What happens if the currently deploying feature is NOT the latest build in Drone?

File: configs/circuit.yml [hunk 3]
```diff
@@ -24,6 +24,7 @@ plugins:
       environments:
         parent: "staging"
         child: "production"
+    internal_bearer_token: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPbmxpbmUgSldUIEJ1aWxkZXIiLCJpYXQiOjE1NjE1MTgwNjMsImV4cCI6MTU5MzA1NDA3MCwiYXVkIjoiY29kZWFtcCIsInN1YiI6IkNnMHdMVE00TlMweU9EQTRPUzB3RWdSdGIyTnIiLCJlbWFpbCI6ImNvZGVhbXBAY29kZWFtcC5vcmciLCJyb2xlIjoibG9jYWwifQ.A_l36r6Nh6-iUTJ2c8OQ0C-4T-ZXmT2CquzultbTk5I"
```
Comment (aballman) [inline configs/circuit.yml]: What's this? This seems like something should not be on github?

File: plugins/scheduledbranchreleaser/scheduledbranchreleaser_test.go [hunk 4]
```diff
@@ -327,17 +326,17 @@ func (suite *TestSuiteScheduledBranchReleaserExtension) TestSBRExtHandlePulseSuc
 		}
 	}
 
-	event := transistor.NewEvent(plugins.GetEventName("heartbeat"), transistor.GetAction("status"), plugins.HeartBeat{Tick: "minute"})
-	suite.transistor.Events <- event
+	// event := transistor.NewEvent(plugins.GetEventName("heartbeat"), transistor.GetAction("status"), plugins.HeartBeat{Tick: "minute"})
```
Comment (aballman) [inline plugins/scheduledbranchreleaser/scheduledbranchreleaser_test.go]: Can you leave a comment as to why this is suddenly commented out, or remove it?

**Pull request merged**